releng/metaborg/releng/cmd.py

```diff
@@ -57,6 +59,9 @@ def get_list(self, key, defaults=None):
 class MetaborgReleng(cli.Application):
   PROGNAME = 'b'
 
+  minimumMvnVersion = version.parse("3.5.4")
+  forbiddenMvnVersions = [version.parse("3.6.1")]
```

Comment:
According to https://github.com/archimatetool/archi/issues/513#issuecomment-558558002, Maven version 3.6.2 could also give problems. I did not test it with this version though

Reply:
Update: tested with 3.6.2, this is also broken (and indeed gives an error related to polyglot, as indicated in the issue I linked above)

Reply:
Thanks! I added 3.6.2 to the list of broken Maven versions.